src/app/page.js

```diff
@@ -1,5 +1,3 @@
-import UserPage from "./user/[_id]/profile/page";
-import AdminPage from "./admin/[_id]/profile/page";
 import GuestHome from "./GuestHome";
```

Comment:
PRAISE: Nice job cleaning up unnecessary imports and other unnecessary code! I think it's smart to try to do as much refactoring and cleanup as possible while working on features!